modules/tinymce/src/plugins/table/test/ts/browser/ApplyCellStyleCommandTest.ts

```diff
@@ -0,0 +1,140 @@
+import { Step, Log, ApproxStructure, Pipeline } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import { Obj } from '@ephox/katamari';
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import SilverTheme from 'tinymce/themes/silver/Theme';
+import * as TableTestUtils from '../module/test/TableTestUtils';
+
+UnitTest.asynctest('browser.tinymce.plugins.table.ApplyCellStyleCommandTest', (success, failure) => {
+  Plugin();
+  SilverTheme();
+
+  TinyLoader.setup((editor: Editor, onSuccess, onFailure) => {
+    const tinyApis = TinyApis(editor);
+
+    const table = '<table style="border-collapse: collapse; width: 100%;" border="1">' +
+      '<tbody>' +
+      '<tr>' +
+      `<td style="width: 50%;" >a</td>` +
+      `<td style="width: 50%;" >b</td>` +
+      '</tr>' +
+      '</tbody>' +
+      '</table>';
+
+    const mapStyles = (styles: Record<string, string>, str) => Obj.map(styles, (val, _key) => str.is(val));
+
+    const sAssertTableCellStructure = (styles: Record<string, string> = {}) =>
+      TableTestUtils.sAssertTableStructure(editor, ApproxStructure.build((s, str, _arr) => s.element('table', {
+        styles: {
+          'width': str.is('100%'),
+          'border-collapse': str.is('collapse')
+        },
+        attrs: {
+          border: str.is('1')
+        },
+        children: [
+          s.element('tbody', {
+            children: [
+              s.element('tr', {
+                children: [
+                  s.element('td', {
+                    styles: {
+                      width: str.is('50%'),
+                      ...mapStyles(styles, str)
+                    },
+                    children: [
+                      s.text(str.is('a'))
+                    ]
+                  }),
+                  s.element('td', {
+                    styles: {
+                      width: str.is('50%'),
+                    },
+                    children: [
+                      s.text(str.is('b'))
+                    ]
+                  })
+                ]
+              }),
+            ]
+          })
+        ]
+      })));
+
+    const sApplyCellStyle = (editor: Editor, args: Record<string, any>) => Step.sync(() => editor.execCommand('mceTableApplyCellStyle', false, args));
```

Comment:
The types here look to be incorrect, as args only ever allows it to be `Record<string, string>`:
```
export type FormatVars = Record<string, string>;
```

Reply:
Thanks for that. This should be fixed now.

Reply:
Looks good :+1: